<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLColorPickerWidget</class>
 <widget class="qMRMLWidget" name="qMRMLColorPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="0" column="0">
    <widget class="QLabel" name="ColorTableLabel">
     <property name="text">
      <string>Color node:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLColorTableComboBox" name="ColorTableComboBox"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="qMRMLColorListView" name="MRMLColorListView">
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkSearchBox" name="SearchBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorListView</class>
   <extends>QListView</extends>
   <header>qMRMLColorListView.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header>ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ColorTableComboBox</sender>
   <signal>currentNodeChanged(vtkMRMLNode*)</signal>
   <receiver>MRMLColorListView</receiver>
   <slot>setMRMLColorNode(vtkMRMLNode*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qMRMLColorPickerWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
